### PR TITLE
Fixed some PHP Code Standards

### PR DESCRIPTION
**Pull request by mjacobus**
I just improved the "new", "def" snippets and fixed some spaces to be PSR-2 compliant.

### CODE REVIEW

File: UltiSnips/php.snippets [hunk 1]
```diff
@@ -8,7 +8,7 @@ $${1:arrayName} = array('${2}' => ${3});${4}
 endsnippet
 
 snippet def "def"
-define('${1}'${2});${3}
+define('${1:FOO}', ${2:'bar'});${3}
```
Comment (lpil) [inline UltiSnips/php.snippets]: Please don't use non-documentary placeholder text.

File: UltiSnips/php.snippets [hunk 2]
```diff
@@ -240,14 +246,14 @@ $0
...
-foreach ($${1:variable} as $${3:value}){
+foreach ($${1:variable} as $${3:value}) {
 	${VISUAL}${4}
 }
 $0
 endsnippet
 
 snippet new "New class instance" b
-$$1 = new $1($2);
+$${1:foo} = new ${2:${1/\w+\s*/\u$0/}}($3);
```
Comment (lpil) [inline UltiSnips/php.snippets]: Here too!

**Comment by lpil** [comment]
See my notes above :)

**Comment by SirVer** [comment]
@lpil Louis, I just learned that snipMate now has a "version 2" file format. I think we should reconsider what lives in `snippets/`, because neither NeoSnippet nor UltiSnips will correctly deal with version 2 snippets. Maybe it is time to split the repo again.

File: UltiSnips/php.snippets (hunk 1 above)
Comment (mjacobus) [inline UltiSnips/php.snippets]: I'm sorry, what is a "non-documentary placeholder text"?

Comment (SirVer) [inline UltiSnips/php.snippets]: FOO, TODO, BLAH, BLUB, /* code */. Just make `${1}`.

**Comment by lpil** [comment]
That's annoying- we're going to have a lot of duplication. Can we have another folder for snipmate v2 snippets?

One for snipmate format, one for Ultisnips format? What format does NeoSnippet use?

Do these repos already exist? How do we communicate this change to users? Seems like it could be a bit of a painful transition.

File: UltiSnips/php.snippets (hunk 1 above)
Comment (mjacobus) [inline UltiSnips/php.snippets]: All right. Fixed. I thought FOOs and BARs were pretty straightforward though. =P

**Comment by SirVer** [comment]
> That's annoying- we're going to have a lot of duplication. Can we have another folder for snipmate v2 snippets?

dunno. I think snipMate did not change their search path. @garbas, can you comment? 

>  What format does NeoSnippet use?

I think it only parses v1 snipMate snippets. @Shougo, can you comment?

**Comment by ajzafar** [comment]
As I stated in #528, this repo need change nothing until contributors decide otherwise.

**Comment by ajzafar** [comment]
Or add `version 0` to the top of SnipMate snippets files.

**Comment by SirVer** [comment]
Reviewed files:
- UltiSnips/php.snippets @ r2

---





Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/honza/vim-snippets/523)

**Comment by SirVer** [comment]
All good. Gonna merge. Thanks!

---









Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/honza/vim-snippets/523)

**Pull request merged**